app/src/features/surveys/observations/sampling-sites/edit/components/SampleSiteEditForm.tsx

```diff
@@ -33,7 +33,7 @@ export interface ISampleSiteEditFormProps {
 
 export const samplingSiteYupSchema = yup.object({
   sampleSite: yup.object({
-    name: yup.string().default(''),
+    name: yup.string().default('').max(50),
```

Comment:
Can you specify a message for the max check (not sure what the default message is, but probably something gross) `'Maximum 50 characters'`